### DISCUSSION TITLE
Can I use hjkl / C-u C-d to scroll of Discord?

### DISCUSSION THREAD

**Discussion by ziontee113** [tridactyl/tridactyl#4631] [Q&A]
Hello. I'm new to Tridactyl and the experience has been pretty amazing so far :+1: 

One little issue I have is that I can't scroll using the movement keys on Discord web page. Is there a way to scroll on Discord using movement keys? Thank you very much!

**Comment by MartinKjarP6N** [comment]
Unfortunately, I don't have enough information to answer your question. Could you please clarify which specific docker version you are referring to? Once I have more context, I'll do my best to provide you with an accurate answer.

**Comment by ziontee113** [comment]
Hi @MartinKjarP6N , I'm sorry for not providing enough information for my question.

On the web app Discord, if I want to use `h j k l` or ` <C-d> <C-u>` to scroll the messages, I'd have to be already focused on the HTML elements of one of the messages.

After chatting with @bovine3dom on Discord, I discovered that I can create special rules to cherry pick elements to jump to / focus to like
```
bindurl www.google.com f hint -Jc .jBgGLd , .OSrXXb .cHaqb , .hdtb-msel , .hdtb-msel+ .hdtb-mitem a , .DKV0Md , .xvfwl , #hdtb-msb a
```

That command works quite well for Google, however I can't seem to make it work. I'm trying
 ```
bindurl discord.com f hint h3
```
but it doesn't seem to give me any hints.
